cde/sdk/go/pkg/cdf/events/extensions/service_extension.go

```diff
@@ -0,0 +1,67 @@
+package extensions
+
+import (
+	"github.com/cloudevents/sdk-go/v2/binding"
+	"github.com/cloudevents/sdk-go/v2/types"
+)
+
+const (
+	ServiceEnvIdExtension   = "serviceenvid"
+	ServiceNameExtension    = "servicename"
+	ServiceVersionExtension = "serviceversion"
+)
+
+// ServiceExtension represents the extension for extension context
+type ServiceExtension struct {
+	ServiceEnvId   string `json:"ServiceEnvId"`
+	ServiceName    string `json:"Servicename"`
```

Comment:
Is the casing different by design? ServiceEnvId in PascalCase and Servicename, Serviceversion in not-PascalCase

Reply:
Nice catch, I'll fix this!